spec/features/preassembly_run/book_using_file_manifest_spec.rb

```diff
@@ -12,7 +12,7 @@
   let(:object_staging_dir) { Rails.root.join(Settings.assembly_staging_dir, 'bb', '000', 'kk', '0000', bare_druid) }
   let(:dro_access) { { view: 'world' } }
   let(:item) do
-    Cocina::RSpec::Factories.build(:dro, type: Cocina::Models::ObjectType.image).new(access: dro_access)
+    Cocina::RSpec::Factories.build(:dro, type: Cocina::Models::ObjectType.book, reading_order: 'right-to-left').new(access: dro_access)
   end
```

Comment:
this spec had the incorrect object type set in the factory (should be "book" for this particular test), which turned out to cause the spec to fail with the new change in this PR